matchms/filtering/filters/harmonize_undefined_template.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+from matchms.filtering.filters.base_spectrum_filter import BaseSpectrumFilter
+from matchms.typing import SpectrumType
+
+
+class HarmonizeUndefinedTemplate(BaseSpectrumFilter):
```

Comment:
Maybe good for file organizations to make a separate folder with these kinds of parent classes. The folder could be called FilterTypes for instance.